adapters/beachfront/beachfront.go

```diff
@@ -663,19 +677,21 @@ func getIP(ip string) string {
 	return ip
 }
 
-func getBidType(externalRequest *adapters.RequestData) openrtb_ext.BidType {
-	t := strings.Split(externalRequest.Uri, "=")[0]
-	if t == videoEndpoint {
-		return openrtb_ext.BidTypeVideo
-	}
-
-	return openrtb_ext.BidTypeBanner
-}
-
 func removeVideoElement(slice []beachfrontVideoRequest, s int) []beachfrontVideoRequest {
 	return append(slice[:s], slice[s+1:]...)
 }
 
-func NewBeachfrontBidder() *BeachfrontAdapter {
-	return &BeachfrontAdapter{}
+func NewBeachfrontBidder(bannerEndpoint string, extraAdapterInfo string) adapters.Bidder {
```

Comment:
You may want to either provide a default video endpoint, or put some checks in to not attempt to bid on video traffic if one is not available. I can see some issues occurring if the adapter is trying to generate requests to a blank URL.

Reply:
I added the default video endpoint URL. Its value is used when the `extraAdapterInfo` is empty or it doesn't contain the value for video endpoint URL.